argo_rollouts/tests/common.py

```diff
@@ -0,0 +1,133 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.dev import get_here
+
+HERE = get_here()
+
+OM_MOCKED_INSTANCE = {
+    'openmetrics_endpoint': 'http://argo_rollouts:8090/metrics',
+    'tags': ['test:tag'],
+}
+
+OM_METRICS = [
+    'argo_rollouts.conroller.info',
+    'argo_rollouts.controller.clientset.k8s.request.count',
+    'argo_rollouts.go.gc.duration.seconds.count',
+    'argo_rollouts.go.gc.duration.seconds.quantile',
+    'argo_rollouts.go.gc.duration.seconds.sum',
+    'argo_rollouts.go.goroutines',
+    'argo_rollouts.go.info',
+    'argo_rollouts.go.memstats.alloc_bytes',
+    'argo_rollouts.go.memstats.alloc_bytes.count',
+    'argo_rollouts.go.memstats.buck_hash.sys_bytes',
+    'argo_rollouts.go.memstats.frees.count',
+    'argo_rollouts.go.memstats.gc.sys_bytes',
+    'argo_rollouts.go.memstats.heap.alloc_bytes',
+    'argo_rollouts.go.memstats.heap.idle_bytes',
+    'argo_rollouts.go.memstats.heap.inuse_bytes',
+    'argo_rollouts.go.memstats.heap.objects',
+    'argo_rollouts.go.memstats.heap.released_bytes',
+    'argo_rollouts.go.memstats.heap.sys_bytes',
+    'argo_rollouts.go.memstats.lookups.count',
+    'argo_rollouts.go.memstats.mallocs.count',
+    'argo_rollouts.go.memstats.mcache.inuse_bytes',
+    'argo_rollouts.go.memstats.mcache.sys_bytes',
+    'argo_rollouts.go.memstats.mspan.inuse_use',
```

Comment:
Is this a typo? I can't seem to find it in the payload...

Reply:
🤦 You're right. I have to go back and fix this in my other integrations as. I was trying to keep the naming universal and copied it over from argocd but it was mistakenly named there. I guess I have to and change it there as well.